macros/GroupData.json

```diff
@@ -1754,31 +1754,30 @@
                             "XRViewerPose",
                             "XRViewport",
                             "XRWebGLLayer" ],
-            "methods":    [ ],
+            "methods":    [ "WebGLRenderingContextBase.makeXRCompatible()" ],
```

Comment:
"WebGLRenderingContextBase" isn't a thing that is exposed to web developers, so we didn't document it this way. I think you want WebGLRenderingContext here.

Reply:
Yes, you're right. Will fix.

Reply:
@Elchi3 The suggestion seems right. Is this the thing you wanted me to look at?

Reply:
This and possible the other changes here, since I'm not a subject matter expert.